app/internal/json_data_loader.py

```diff
@@ -54,3 +54,25 @@ def load_to_db(session) -> None:
     """The quotes JSON file content is copied from the free API:
     'https://type.fit/api/quotes'. I saved the content so the API won't be
     called every time the app is initialized."""
+    load_data(
+        session, 'app/resources/parashot.json',
+        Parasha, weekly_parasha.create_parasha_object)
+    """The quotes JSON file content is copied from the free API:
+    'https://www.hebcal.com/hebcal?v=1&cfg=json&maj=on&min=on&
+    mod=on&nx=on&year=now&month=x&ss=on&mf=on&c=on&geo=geoname
+    &geonameid=293397&m=50&s=on&d=on&D=on'.
+    Each year the Json file will need to be re-updated
+    according to this API in this way:
+
+    def relevent_details(p):
+    parashot_dict = {'name': p['title'], 'hebrew': p['hebrew'],
+     'link': p['link'], 'date': p['date']}
+    return parashot_dict
+
+
+    def get_all_parashot(API):
+    r = requests.get(API)
+    items = r.json()['items']
+    all_para = list(filter(lambda i: 'Parashat' in i['title'], items))
+    return [relevent_details(p) for p in all_para]
```

Comment:
A few notes here:
* Move this to the parashot module as it has nothing to do with loading from JSON. 
* Don't use "p" or "r" as variable names.
* The "API" parameter in the function should be "api".

Regarding
`all_para = list(filter(lambda i: 'Parashat' in i['title'], items))`
`return [relevent_details(p) for p in all_para]`

Can't that be done with a simple if check in the list comprehension?
`[relevent_details(p) for p in all_para if 'Parashat' in p['title]]`?

Reply:
i will do all the changes.thanks :)
By the way, can you help me with an error that comes back to me from the PYTEST test?